src/helpers/figures/charts/bar_chart.ts

```diff
@@ -304,7 +304,10 @@ export function createBarChartRuntime(chart: BarChart, getters: Getters): BarCha
       label,
       data,
       borderColor: BORDER_CHART_COLOR,
-      borderWidth: 1,
+      borderWidth: {
+        bottom: index === "0" || !chart.stacked ? 0 : 2,
+      },
+      borderSkipped: false,
```

Comment:
Thinking about it a bit more, it unfortunately doesn't work 😞 If the stacked bar chart contain negative values, we want the top border, not the bottom.

I'd say we should probably either enable all the borders of the stacked bar charts, or disable all the bar borders always and not bother too much.